static/index.html

```diff
@@ -6,6 +6,7 @@
     <title>PictoPy</title>
     <link rel="icon" href="{{ url_for('static', filename='asset/favicon.ico') }}">
     <link rel="stylesheet" href="/static/index.css"/>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
```

Comment:
we cant have this due to offline nature :(

For icons I would prefer svgs saved locally. 
You can check https://www.svgrepo.com.

Reply:
Different tones of black or white are fine, but please refrain from using other colors for now.

Reply:
Is this okay, I have used the color from previous defined color in css file 


![Screenshot from 2024-08-13 23-56-48](https://github.com/user-attachments/assets/aa0207a3-1037-4f67-b413-caf899e63523)

Reply:
that looks good